_data/devdocs/en/guides/wallets.md

```diff
@@ -341,18 +341,18 @@ address utility].
 
 {% autocrossref %}
 
-Bitcoin ECDSA public keys represent a point on a particular Elliptic
-Curve (EC) defined in secp256k1. In their traditional uncompressed form,
-public keys contain an identification byte, a 32-byte X coordinate, and
+Bitcoin ECDSA public key represents a point on a particular Elliptic
```

Comment:
This change would require an article to be inserted at the beginning of the sentence ([A] Bitcoin ECDSA public key represents....).